db/migrate/20170109221226_create_configuration_projects.rb

```diff
@@ -0,0 +1,17 @@
+class CreateConfigurationProjects < ActiveRecord::Migration[5.0]
+  def change
+    create_table :configuration_projects do |t|
+      t.belongs_to  :configuration_manager, :type => :bigint
+      t.string      :manager_ref
+      t.string      :name
+      t.string      :description
+      t.string      :scm_type
+      t.bigint      :scm_reference_id
```

Comment:
columns that aren't foreign keys should never end in `_id` either.  Please consider `scm_ref` (to mirror ems_ref naming) if you still need it